src/core/qgsvectorfilewriter.h

```diff
@@ -576,6 +580,8 @@ class CORE_EXPORT QgsVectorFileWriter : public QgsFeatureSink
      * \param layerName layer name. If let empty, it will be derived from the filename (added in QGIS 3.0)
      * \param action action on existing file (added in QGIS 3.0)
      * \param newLayer potentially modified layer name (output parameter) (added in QGIS 3.4)
+     * \param transformContext transform context, needed if the output file srs is forced to specific crs (added in QGIS 3.10.3)
+     * \param sinkFlags feature sink flags (added in QGIS 3.10.3)
      * \note not available in Python bindings
      */
     QgsVectorFileWriter( const QString &vectorFileName,
```

Comment:
This should be deprecated too

Reply:
I initially did, but deprecating that constructor leads to a warning when we *have* to use it in our static create function. Solution?

Reply:
Wrap the call in a NOWARN block -- I don't think we can cleanly fix this whole class until 4.0 when we can hard break it